timemachine/lib/potentials.py

```diff
@@ -4,8 +4,6 @@
 # (ytz): classes in this class wrap custom_ops but have the added benefit
 # of being pickleable.
 
-BoundPotential = custom_ops.BoundPotential
```

Comment:
Is the removal of this alias intentional?

Reply:
Woops

Reply:
Actually looks like it isn't used. I had removed it to be able to run the `generate_star_map.py`. Do we use this alias?

Reply:
I think it's safe to drop this then. Was the idea that generate_star_map can be ran entirely without the custom_ops?

Reply:
I think at one point we had an instance(potentials.BoundPotential as an alias) - but its probably better to just directly reference the class in the custom_ops to make parts of the code more portable.

Reply:
This was so I could run it on my mac without it, though hadn't expected to keep this change. Since it looks unused, I agree that we should reference classes via `custom_ops`